app/controllers/application_controller.rb

```diff
@@ -16,4 +16,12 @@ def authorize_user
       raise ActionController::RoutingError.new("Not Found")
     end
   end
+
+  def after_sign_in_path_for(user)
```

Comment:
Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. You can also write as `after_sign_in_path_for(*)` if you want the method to accept any arguments but don't care about them.